guide/verifying-installation.rst

```diff
@@ -28,65 +27,127 @@ time.
-Usually, we launch MATLAB from the ``DART/models/lorenz_63/work``
-directory and use the MATLAB *addpath* command to make the
-``DART/matlab/`` functions available for execution in any working
-directory.
+Usually, we launch MATLAB from the ``DART/models/lorenz_63/work`` directory.
+Execute the following command in your terminal to navigate to this directory.
+
+.. code-block:: bash
+
+   $ cd DART/models/lorenz_63/work
+
+.. note:: The lines of code that are marked with a ``$`` symbol are commands that
+          you will execute in your terminal. To execute a command, type it in
+          exactly as seen, unless specified otherwise, and hit enter.
 
 In the case of this Lorenz model, we know the "true" (by definition)
-state of the model that is consistent with the observations, which was
-generated by the *perfect_model_obs* program as described in `Checking
-the build — running something <#runningSomething>`__. The following
-MATLAB scripts compare the ensemble members with the truth and can
-calculate the error in the assimilation:
+state of the model that is consistent with the observations, which is
+generated by running the ``perfect_model_obs`` program. We therefore must first
+run ``perfect_model_obs`` before launching MATLAB and running our diagnostic
+scripts. To do so, execute the following three commands in the terminal.
 
 .. code-block:: bash
 
-   $ cd DART/models/lorenz_63/work
-   $  matlab -nodesktop
-   (Skipping startup messages)
-
-       [matlab_prompt] addpath ../../../diagnostics/matlab
-       [matlab_prompt] plot_total_err
-       Input name of true model trajectory file;
-       (cr) for perfect_output.nc
-       perfect_output.nc
-       Input name of ensemble trajectory file;
-       (cr) for preassim.nc
-       preassim.nc
-       Comparing true_state.nc and
-                 preassim.nc
-       [matlab_prompt] plot_ens_time_series
-       Input name of ensemble trajectory file;
-       (cr) for preassim.nc
-
-       Comparing true_state.nc and
-                 preassim.nc
-       Using Variable state IDs 1  2  3
-
-       pinfo =
-
-         struct with fields:
-
-                        model: 'Lorenz_63'
-                      def_var: 'state'
-               num_state_vars: 1
-                   num_copies: 20
-              num_ens_members: 20
-             ensemble_indices: [1 2 3 ... 18 19 20]
-                min_state_var: 1
-                max_state_var: 3
-               def_state_vars: [1 2 3]
-                        fname: 'preassim.nc'
-                   truth_file: 'true_state.nc'
-                   diagn_file: 'preassim.nc'
-                   truth_time: [1 200]
-                   diagn_time: [1 200]
-                         vars: {'state'}
-                         time: [200x1 double]
-           time_series_length: 200
-                          var: 'state'
-                     var_inds: [1 2 3]       
+   $ csh quickbuild.csh
+   $ ./filter
+   $ ./perfect_model_obs
```

Comment:
perfect_model_obs creates observations to be used by filter, so although this order will work as is, it is
better for people's understanding of DART to switch the order of these commands

```suggestion
   $ ./perfect_model_obs
   $ ./filter
```